modules/network/vpc/variables.tf

```diff
@@ -118,44 +122,52 @@ variable "primary_subnetwork" {
   type        = map(string)
   default     = null
   validation {
-    condition     = var.primary_subnetwork == null || can(var.primary_subnetwork["subnet_name"])
+    condition     = var.primary_subnetwork == null ? true : can(var.primary_subnetwork["subnet_name"])
     error_message = "Primary subnetwork must define \"subnet_name\"."
   }
   validation {
-    condition     = var.primary_subnetwork == null || can(var.primary_subnetwork["subnet_region"])
```

Comment:
I guess all those old validation blocks can be simply removed.

Reply:
Done!